config/routes.rb

```diff
@@ -1157,8 +1164,6 @@
         protect
         show_list
         tagging_edit
-        scaling
```

Comment:
I thought I'd removed scaling from ems_physical_infra. As of yet it's unknown how we can scale up hardware.